### PR TITLE
Enhancements to AsciiDoc snippets for Sublime

### PR DESCRIPTION
**Pull request by polyglot-jones**
Changed one snippet and added three more, taking full advantage of Sublime's powerful snippet features.
Added an AsciiDoc spec for the build system.
Updated the Readme to mention all of the above.

### CODE REVIEW

**Comment by gAmUssA** [comment]
@FichteFoll any chance this PR will be merged? it has some nice snippets

**Comment by FichteFoll** [comment]
The general changes are good, but I consider the enhancements to involve too much user interaction and they won't fare well like this on ST3 due to archived packages.

I have plans to improve the PR in that regard and will merge it along with a few changes when I find the time.

File: Article.sublime-snippet [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<snippet>
+   <description>AsciiDoc Article Start</description>
+   <content><![CDATA[${1:Title}
+${1/./=/g}
+$TM_FULLNAME <$TM_EMAIL>
+v1.0, $TM_YEAR
+:copyright: (c) $TM_ORGANIZATION_NAME, $TM_YEAR. All rights Reserved.
+:backend: ${2:html5|wordpress|slidy}
```
Comment (jrappen) [inline Article.sublime-snippet]: `${2:html5|wordpress|slidy}` simply gives me `html5|wordpress|slidy`, is this intended?

Comment (FichteFoll) [inline Article.sublime-snippet]: I suppose this is to make the user choose one of these by showing which are "available" (or just showing an example). I say it's intended.

Comment (polyglot-jones) [inline Article.sublime-snippet]: Yeah, that was the closest I could get to a drop-down box.

**Comment by aggsol** [comment]
Anything will happen here?

**Comment by FichteFoll** [comment]
Since some people seem to care about it, I will just merge it, although I consider it quite unmaintainable in its current form. If any further PRs arise that improve on this situation, I will happily merge them as the package is obviously abandoned, but I don't see myself doing maintenance myself unless I somehow end up using it.

**Pull request merged**